src/client/Microsoft.Identity.Client/PlatformsCommon/Shared/SimpleHttpClientFactory.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Net.Http;
+using Microsoft.Identity.Client.Http;
+
+namespace Microsoft.Identity.Client.PlatformsCommon.Shared
+{
+    /// <summary>
+    /// A simple implementation of the HttpClient factory that uses a managed HttpClientHandler
+    /// </summary>
+    /// <remarks>
+    /// This implementation is not suitable for high-scale applications / confidential client scenarios
+    /// because creating new HttpClients leads to a port exhaustion issue. 
+    /// Mobile platforms should use HttpClientHandlers that are platform specific.
+    /// See platform specific implementations for details on these issues.
+    /// </remarks>
+    internal class SimpleHttpClientFactory : IMsalHttpClientFactory
```

Comment:
We would still use this on Mac and NetStandard, but on Mac we only have PublicClient and we don't recommend pure NetStandard  implementation